share/gpodder/ui/gtk/menus.ui

```diff
@@ -267,5 +267,51 @@
       </item>
     </section>
   </menu>
+  <menu id="episodes-context">
+      <item>
```

Comment:
Should these items be wrapped in a `<section>`?

Reply:
Yes, it does not seem to affect how the menu looks like though.

Reply:
I forgot to say the download menu was also missing a `<section>`. They don't change the look, but they are probably needed to add buttons above the items.

Reply:
Added, thanks.